mlflow/metrics/genai/genai_metric.py

```diff
@@ -220,6 +221,8 @@ def make_genai_metric_from_prompt(
         metric_metadata: (Optional) Dictionary of metadata to be attached to the
             EvaluationMetric object. Useful for model evaluators that require additional
             information to determine how to evaluate this metric.
+        genai_metric_args: (Optional) A dictionary containing arguments required to call
+            make_genai_metric_from_prompt for restoring the metric instance later
```

Comment:
```suggestion
```
docstring is for the function arguments so we don't need to add it here:)

Reply:
ohh shoot I shouldn't be working at night.... my brain isn't braining. thanks for catching it up